clar2wasm/tests/wasm-generation/control_flow.rs

```diff
@@ -1,19 +1,19 @@
 use clar2wasm::tools::crosscheck;
 use clarity::vm::Value;
-use proptest::prelude::prop;
-use proptest::proptest;
+use proptest::prelude::{prop, proptest};
 use proptest::strategy::Strategy;
 
 use crate::PropValue;
 
-fn begin_strategy() -> impl Strategy<Value = (String, PropValue, bool)> {
-    prop::collection::vec(PropValue::any(), 1..=10).prop_map(|values| {
+fn generate_begin_expressions(
+    maximum_expressions: usize,
+) -> impl Strategy<Value = (String, PropValue, bool)> {
+    prop::collection::vec(PropValue::any(), 1..=maximum_expressions).prop_map(move |values| {
```

Comment:
If this is used only for `begin`, shouldn't this function be inlined in the begin test? That's what I said in the previous PR [here](https://github.com/stacks-network/clarity-wasm/pull/424#discussion_r1642510712)

Reply:
I got the idea that you're only interested in name change. I'm not sure what do you think of lengthy functions and separation of concerns. What I thought that building expressions isn't the test's job. I know it's not being used anywhere else but aren't they only tradeoffs?

Reply:
What separation of concern are you talking about here? it literally
```rust
fn _foo() {
    // super 
    // long 
    // body
}

pub fn foo() {
    _foo()
}
```

Reply:
Alright. How do you make a longer function readable? I mean there are examples which splits a complex function into smaller functions just to make code readable. Those functions aren't used anywhere. There is one problem with this though. Users debugging the code might lose context if they are too many.

Reply:
A 20 lines function should normally be readable, nothing special. You can always add comments to describe each sections of your functions if you feel like it could make the reading easier.